docs/introduction.md

```diff
@@ -1,29 +1,51 @@
 ---
-title: What is Quickwit?
+title: Introduction
 slug: /
 sidebar_position: 1
 ---
 
-Quickwit is a distributed search engine built from the ground up to offer cost-efficiency and high reliability. By mere mortals for mere mortals, Quickwit's architecture is as simple as possible[^1].
+Quickwit is a distributed search engine designed from the ground up to offer cost-efficiency and high reliability on large data sets.
 
-Quickwit is written in Rust and built on top of the mighty [tantivy](https://github.com/tantivy-search/tantivy) library. We designed it to index large datasets.
+Quickwit is particularly well-suited for dealing with large, immutable datasets and a low average QPS$^1$. Its benefits are most apparent in a multi-tenancy or multi-index setting.
 
-## Why Quickwit?
+Common use cases for Quickwit include:
 
-Quickwit is born from the idea that today's search engines are hard to manage and uneconomical when dealing with large datasets and a low QPS[^2] rate. Its benefits are most apparent in a multitenancy or a multi-index setting.
+- Searching through logs, from a small amount to TB of data.
+- Adding full-text search capabilities to [OLAP databases such as ClickHouse](link to blog post).
+- Searching through backups sitting on your cloud storage by adding Quickwit index files on your same storage.
```

Comment:
This needs to be reworded but I have a hard time coming up with something compelling.

Reply:
Broken link

Reply:
Link fixed.